docs/Configuration.md

```diff
@@ -55,6 +62,20 @@ Alternately you can set the following environment variables. If any of these are
   DSPACE_REST_NAMESPACE=/server
 ```
 
+## Environment variable naming convention
+
+Settings can be set using the following convention for naming the environment variables:
+
+1. replace all `.` with `_`
+2. convert all characters to upper case
+
+e.g.
+
+```
+cache.msToLive.default => CACHE_MSTOLIVE_DEFAULT
+auth.ui.timeUntilIdle => AUTH_UI_TIMEUNTILIDLE
```

Comment:
These examples need to be updated as well to prefix with `DSPACE_` & update rules on lines 69 & 70.

Reply:
Updated. Thanks.